### ISSUE TITLE
Various a11y updates

### ISSUE THREAD

**Issue by Tetsuro** [Tetsuro/tetchi-blog#16]
- [x] In this case I'd have a `visually-hidden` title, "tetchi's blawg". Then each post title would be an `h2`.
- [x] Also, your "read more" links should have `visually-hidden` text to give more context.
- [x] each `class=post` could be an `article` element, and the `.content__wrapper` could be a `main`.
- [x] You could also add either `visually-hidden` or `aria-label` to your footer icon links. Right now there's no accessible name.

Feedback from Scott Vinkle :heart:

**Comment by Tetsuro** [comment]
CLOSED
